docs/apache-airflow/plugins.rst

```diff
@@ -167,9 +172,14 @@ definitions in Airflow.
 
     # Importing base classes that we need to derive
     from airflow.hooks.base import BaseHook
+    from airflow.listeners.listener import Listener
     from airflow.models.baseoperator import BaseOperatorLink
     from airflow.providers.amazon.aws.transfers.gcs_to_s3 import GCSToS3Operator
 
+    from pluggy import HookimplMarker
+
+    hookimpl = HookimplMarker("airflow")
```

Comment:
Looking at the pluggy docs it appears the normal pattern here is to import the hookimpl from a module rather than to create it here: https://pluggy.readthedocs.io/en/latest/#the-plugin

Reply:
Done.